anomalyLanguages.js

```diff
@@ -0,0 +1,20 @@
+const anomalyMessages = {
```

Comment:
Nice way to isolate the languages into one place.
We need a way to mistake-proof this: Consider we are adding another language, like Spanish. A developer may forget to add the Spanish text on one of these rows.
Think of a test to catch such a condition. Hint: Design the test at a lower level (such as the contents of this object, or `getAnomalyMessage`, don't go for testing every condition in every language.